__tests__/index.test.js

```diff
@@ -254,7 +275,127 @@ describe('/api/articles/:article_id/comments', () => {
             })
     });
 
- })
+    test('sends an appropriate status and error message when body field is missing', () => {
+
+        const newComment = {username: 'icellusedkars'}
+
+        return request(app)
+           .post('/api/articles/9/comments')
+            .send(newComment)
+            .expect(400)
+            .then((response) => {
+                expect(response.body.msg).toBe("missing field - body");
+            });
+    });
+
+})
+
+describe('PATCH /api/articles/:article_id', () => {
+    test('PATCHES an article with an updated votes property - status 201', () => {
+
+        const voteUpdate = { inc_votes: 5 }
+
+        const returnedArticle = {
```

Comment:
If you wanted to clean up the testing you could assert that the votes has the **correct updated value** and, rather than having to have these returned article variables, just assert that the properties are there and are the correct data type.